src/views/components/debug-display/index.css

```diff
@@ -0,0 +1,8 @@
+::-webkit-scrollbar {
+  -webkit-appearance: none;
+  width: 4px;
+}
+::-webkit-scrollbar-thumb {
+  background-color: rgba(0, 0, 0, 0.5);
```

Comment:
These CSS is there so that user sees the scroll bar while viewing the JSON data, otherwise there would be no scroll bar and user could get confused thinking there is no data.